flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ArchivedExecutionGraphTest.java

```diff
@@ -142,6 +143,8 @@ public static void setupExecutionGraph() throws Exception {
 
 		Execution executionWithAccumulators = runtimeGraph.getJobVertex(v1ID).getTaskVertices()[0].getCurrentExecutionAttempt();
 
+		runtimeGraph.setJsonPlan("{}");
```

Comment:
this seems unrelated. Probably part of porting the JobPlanHandler?

Reply:
This PR introduces null checks in the `ArchivedExecutionGraph` and that's why this change is included. I could also move these changes to the #4727 PR.

Reply:
ah ok, nevermind then.